clocksilencer.ino

```diff
@@ -99,5 +99,5 @@ void ringer(int myhours2) {
     Serial.println(n);
     delay(2250);
   }
-  myDFPlayer.sleep();
+  //myDFPlayer.sleep();
```

Comment:
Why is this comented? Don't we need DFPlayer to go into deep sleep?

Reply:
This causes troubles in waking up the Player.

Reply:
ok, let's leave it for now. we can investigate it later...